src/media/js/categories.js

```diff
@@ -3,23 +3,30 @@ define('categories', ['core/l10n'], function(l10n) {
     var gettext = l10n.gettext;
 
     return [
-        {slug: 'games', name: gettext('Games')},
```

Comment:
It looks like they are alphabetical except for games so I added the new ones in their spot keeping games at the top.